portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase

```diff
@@ -0,0 +1,385 @@
+			}
+		}
+
+		task ("Connect AC and DXP") {
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("Visit the site page") {
+			ACUtils.navigateToSitePageWithUser(
+				emailList = ${emailList},
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Create a data report using analytics cloud API with the type individuals") {
+			var apiResponse = ACApi.createDataReport(
+				token = ${token},
+				type = ${type});
+		}
+
+		task ("Assert the two individuals are present in the API response") {
+			ACApi.assertApiResponse(
+				apiDataResponse = ${apiResponse},
+				expectedList = ${userNameList});
+		}
+	}
+
+	@description = "Feature ID: LRAC-12919 | Automation ID: LRAC-13503 | Test Summary: Check if you can access data for a specific blog"
+	@priority = 3
+	test RequestToAccessDataForSpecificBlog {
+		var blogsTitleList = "Blog AC Title 1,Blog AC Title 2";
+		var type = "blogs";
+		var range = "rangeKey";
+		var rangeValue = 0;
+
+		task ("Copy the token value for the API") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Take the token from Analytics Cloud") {
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Go to DXP and create a site and two blogs and then connect Analytics Cloud to DXP") {
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+
+			ACUtils.addSiteAndPage();
+
+			for (var blogTitle : list ${blogsTitleList}) {
+				ACUtils.createBlogsAndAddToPage(entryTitle = ${blogTitle});
+			}
+
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("Visit the two blogs") {
+			ACUtils.navigateToSitePage(
+				actionType = "View Blog",
+				documentTitleList = "Blog AC Title 1,Blog AC Title 2",
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("Create a data report using analytics cloud API with the type blogs") {
+			var apiResponse = ACApi.createDataReport(
+				filter = ${range},
+				filterValue = ${rangeValue},
+				token = ${token},
+				type = ${type});
+		}
+
+		task ("Assert the total items of the response is 2 (because we have just 2 blogs)") {
+			var totalValue = ACApi.searchApiResponse(
+				apiDataResponse = ${apiResponse},
+				jsonQuery = "total");
+
+			TestUtils.assertEquals(
+				actual = ${totalValue},
+				expected = 2);
+		}
+
+		task ("Get the id of the blog in the first position of the response") {
+			var blogId = ACApi.searchApiResponse(
+				apiDataResponse = ${apiResponse},
+				jsonQuery = ".id",
+				resultPosition = 0);
+		}
+
+		task ("Create a data report using analytics API for the specific blog chosen") {
+			var apiAssetResponse = ACApi.createIndividualOrAssetDataReport(
+				id = ${blogId},
+				token = ${token},
+				type = ${type});
+		}
+
+		task ("Assert that there's only one result in the response") {
+			var searchedValue = ACApi.searchApiResponse(
+				apiDataResponse = ${apiAssetResponse},
```

Comment:
Hey @manuferreira, `apiIndividualOrAssetResponse` is a `static var` so you can use this variable here in your macro and you don't need to pass a new variable with the value of `apiIndividualOrAssetResponse` in the parameter of this macro